oak/server/rust/oak_runtime/src/io.rs

```diff
@@ -83,16 +84,16 @@ pub trait ReceiverExt<T> {
 impl<T: Decodable> ReceiverExt<T> for Receiver<T> {
     /// Close the underlying channel handle.
     fn close(self, runtime: &RuntimeProxy) -> Result<(), OakStatus> {
-        runtime.channel_close(self.handle)
+        runtime.channel_close(self.handle.handle)
```

Comment:
Why does handle have a handle inside?

Reply:
`ReadHandle` and `WriteHandle` are wrappers for the raw handles that indicate the intended use.